internal_transfers/actions/src/orderbook.ts

```diff
@@ -0,0 +1,69 @@
+import { WinningSettlementData } from "./models";
+import axios from "axios";
+
+// https://api.cow.fi/docs/#/
+const PROD_API_URL =
```

Comment:
As a comment, although I guess you are already aware, there are rate limits for the API. Not sure if that can potentially create issues.

Reply:
I would hope that the rate limit is not more than how frequently settlements are happening.

Reply:
Yeah, I would hope so as well, so maybe we don't need to worry about this after all.

Reply:
Generally rate limits are more on par with numbers like 10 per second, so I think we're ok here. However you might have issues with the EBBO thing if you're gonna wake up and make a bunch of requests every 30 minutes.